### PR TITLE
fix(projects): reset values when reapplying the filter

### PR DESCRIPTION
**Pull request by aziiee**
Project and task should be reset when the customer changes. Task
should be reset when the project changes.

### CODE REVIEW

File: app/projects/template.hbs [hunk 1]
```diff
@@ -12,7 +12,7 @@
         options=customers
         placeholder="Select customer..."
         searchField="name"
-        onchange=(action (queue (action (mut selectedCustomer)) (action (mut selectedProject) null) (perform filterProjects)))
+        onchange=(action (queue (action (mut selectedCustomer)) (action (mut selectedProject) null) (action (mut selectedTask) null) (perform filterProjects)))
```
Comment (anehx) [inline app/projects/template.hbs:15]: Please make an action in the controller for this. It's obviously not readable..

File: app/projects/template.hbs [hunk 2]
```diff
@@ -26,7 +26,7 @@
         options=filteredProjects
         placeholder="Select project..."
         searchField="name"
-        onchange=(action (queue (action (mut selectedProject)) (perform fetchTasksOfProject)))
+        onchange=(action (queue (action (mut selectedProject)) (action (mut selectedTask) null) (perform fetchTasksOfProject)))
```
Comment (anehx) [inline app/projects/template.hbs:29]: Same here

File: app/projects/template.hbs (hunk 1 above)
Comment (aziiee) [reply app/projects/template.hbs]: Done

File: app/projects/template.hbs (hunk 2 above)
Comment (aziiee) [reply app/projects/template.hbs]: Done

**Review by anehx — APPROVED** [review]

**Pull request merged**